pkg/web/router.go

```diff
@@ -52,6 +55,7 @@ func NewRouter(config *RouterConfig, wpool pool.WorkerPoolInterface) http.Handle
 	idpConfig := config.idp
 	schemasConfig := config.schemas
 	rulesConfig := config.rules
+	uiConfig := config.ui
```

Comment:
a couple of lines above we could create the router and then have 
https://pkg.go.dev/github.com/go-chi/chi/v5#Mux.Group to create another inline router for the UI

this would avoid us changing all the `api` prefixes